ui/components/suite.jsx

```diff
@@ -1,8 +1,29 @@
-var React = require('react');
+var React = require('react'),
+    Test = require('./test.jsx');
 
 var Suite = React.createClass({
+  statics: {
+    MARGIN_LEFT: 20,
+    FONT_SIZE: 100
+  },
   render: function() {
-    return null;
+    var suite = this.props.suite,
+        suiteStyle = {
+          marginLeft: this.constructor.MARGIN_LEFT * (suite.indent - 1) + 'px'
```

Comment:
![image](https://cloud.githubusercontent.com/assets/11819686/10163582/85eb7f32-66bc-11e5-984f-24682e3c0a20.png)  neither in the browser